src/design-tokens/core-tokens.json

```diff
@@ -1,5 +1,1025 @@
 {
   "eds": {
+    "anim": {
```

Comment:
moving the previous primitives JSON into a renamed core-tokens.json file, to allow ease of updating